_plugins/content_code.rb

```diff
@@ -0,0 +1,24 @@
+require 'rouge'
+
+# Include tabindex for accessibility reasons
+class Rouge::Formatters::HTMLPygmentsA11y < Rouge::Formatters::HTMLPygments
+  def stream(tokens, &b)
+    yield %(<div class="highlight"><pre class="#{@css_class}"><code tabindex="0">)
```

Comment:
As discussed when we paired, I personally think it'd be better to put the tabindex on the element with the `overflow` styling, which also helps avoid some strange focus outline wrapping visuals.

```suggestion
    yield %(<div class="highlight"><pre class="#{@css_class}" tabindex="0"><code>)
```

Reply:
I can resolve this with `code { display: inline-block; }`. This lets me keep the focus on the proper element.

Reply:
Can you elaborate on "proper element"? The focus should be on the scrollable element, which is the `<pre>`.

Reply:
Is the `pre` the scrollable element because it has set dimensions?  
I tested this, and you are correct: the `pre` element is the one that is scrolling. That's not intuitive to me, because the `code` element is the one with content. I guess if it had a text node I wouldn't think that way.

I'll go ahead and make the change you suggested.

Reply:
It's because [the `pre` is the element that has the `overflow` styling](https://github.com/18F/identity-design-system/blob/main/src/scss/packages/usa-code/src/_styles.scss#L35-L37), where the overflow styling will cause scroll if needed ([ref](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x#auto)).